app/services/validation_beta_service.rb

```diff
@@ -22,6 +22,14 @@ def invite_schools(array_of_urns:, participant_research:, coordinator_research:)
     end
   end
 
+  def notify_induction_coordinators_about_validation
+    School.eligible.find_each do |school|
```

Comment:
And cip schools?

Reply:
Do you mean cip-only schools? They won't have added any participants as they aren't invited yet right? And we are emailing them separately later in the week

Reply:
I think what @ethanmills means is that this will send an email to CIP schools?

Reply:
The query has changed quite a bit now to loop over induction coordinators to make sure we just send one email each. 
https://github.com/DFE-Digital/early-careers-framework/pull/938/files#diff-4c570b8127fbf99f726f6b1a409d563cc29b16e9c464c92b90173c71b58ef416R25-R31
I didn't know we wanted to make a distinction between CIP or FIP schools, just that they had chosen a programme, do we need to?

Reply:
Nope, I meant cip-only schools. If we're targeting them separately then that's fine